### ISSUE TITLE
should serviceaccounts be reported even if they don't exist?

### ISSUE THREAD

**Issue by lucasreed** [FairwindsOps/rbac-lookup#37]
TLDR; the service accounts reported as having access may not actually exist. BUT, that might be ok.

Here's a situation to back up my findings:

- RBAC ClusterRoleBinding gives access to two service accounts:
  - `helm-system:tiller`
  - `kube-system:tiller`

```
kubectl get clusterrolebinding tiller -o yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"rbac.authorization.k8s.io/v1beta1","kind":"ClusterRoleBinding","metadata":{"annotations":{},"creationTimestamp":null,"name":"tiller"},"roleRef":{"apiGroup":"rbac.authorization.k8s.io","kind":"ClusterRole","name":"cluster-admin"},"subjects":[{"kind":"ServiceAccount","name":"tiller","namespace":"helm-system"},{"kind":"ServiceAccount","name":"tiller","namespace":"kube-system"}]}
  creationTimestamp: "2019-08-09T15:16:49Z"
  name: tiller
  resourceVersion: "7565"
  selfLink: /apis/rbac.authorization.k8s.io/v1/clusterrolebindings/tiller
  uid: b577bb52-bab8-11e9-b453-0a5bfa843bc0
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: cluster-admin
subjects:
- kind: ServiceAccount
  name: tiller
  namespace: helm-system
- kind: ServiceAccount
  name: tiller
  namespace: kube-system
```

- `helm-system:tiller` exists:

```
kubectl get sa tiller -n helm-system
NAME     SECRETS   AGE
tiller   1         57d
```

- `kube-system:tiller` does not exist:
```

kubectl get sa tiller -n kube-system
Error from server (NotFound): serviceaccounts "tiller" not found
```

- rbac-lookup reports both:
```
rbac-lookup -o wide tiller
SUBJECT                              SCOPE          ROLE                        SOURCE
ServiceAccount/helm-system:tiller    cluster-wide   ClusterRole/cluster-admin   ClusterRoleBinding/tiller
ServiceAccount/kube-system:tiller    cluster-wide   ClusterRole/cluster-admin   ClusterRoleBinding/tiller
```

A part of me thinks this is ok because if `kube-system:tiller` is ever created it will gain this access. It may also be good to include this output so an administrator can go cleanup the clusterrolebinding.

Another part of me thinks this output is not helpful because it is not reporting the exact state of what's present in the cluster.

Wanted to open this issue for discussion and see what other folks think.
